### PR TITLE
updated_form

### LINKED ISSUE #2723: Replacing radio button to dropdown list in service-mesh-management/meshmap form
**Current Behavior**

In the form "What role best identifies you?" in this section 
there is a radio button that occupies a lot of space  
Screenshot from 2022-04-19 18-10-08

**Desired Behavior**
The radio-button should be replace by drop-down button

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Akshit42-hue**
Signed-off-by: Axit Patel <patelakshit2025@gmail.com>

This PR fixes #2723 

**Notes for Reviewers**
I have replace radio button with dropdown-list in form

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Akshit42-hue** [comment]
Hey @debo19 please do have look

**Review by asubedy — CHANGES_REQUESTED** [review]
The size of the input boxes above and the drop down are not same please make them the same size. And also the alignment of the dropdown seems to be off by some pixels please fix that too.

**Comment by Akshit42-hue** [comment]
Thanks for the update @asubedy

**Comment by Akshit42-hue** [comment]
Required changes have been done please do have a look @asubedy

**Comment by l5io** [comment]
🚀 Preview for commit 247f7de913425701c732c8578c2cd9b92291f097 at: https://6262d42f30f632008d5c46f5--layer5ng.netlify.app

**Review by YashKamboj — CHANGES_REQUESTED** [review]
Great work @Akshit42-hue !! just one small thing when nothing is chosen , the dropdown shows "architect", it should have something like choose or select one, you can do it by adding this on the top of the option list : 
`<option value="" disabled>Select one--</option> `

**Comment by Akshit42-hue** [comment]
Thanks @YashKamboj I was searching this, but can't able to find any leads

**Comment by Akshit42-hue** [comment]
Hey @YashKamboj, @asubedy please do have a look

**Comment by l5io** [comment]
🚀 Preview for commit fa01d5063f0133178ae6f549e9f905cf3783b58f at: https://62637b7dedbd5751a9ea10bb--layer5ng.netlify.app

**Review by YashKamboj — APPROVED** [review]

**Review by Abhijay007 — APPROVED** [review]

**Review by debo19 — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/85789734/164949944-b9ebf037-2d73-4790-90d7-216e6f32ea86.mov

@Akshit42-hue On selecting any option, the page refreshes due to which we lose all previous input values.

**Comment by Akshit42-hue** [comment]
Nice catch @debo19, I apologize I need to check all those things before raising a pr

**Comment by Akshit42-hue** [comment]
Required changes have been done, please do have a look @debo19

**Comment by l5io** [comment]
🚀 Preview for commit 1c4fd93fd288fe6fc89b39d9ca2e43721e5415b3 at: https://6264c94adf1bec49cf71dd4d--layer5ng.netlify.app

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

**Comment by debo19** [comment]
@Akshit42-hue lets discuss this on the call today.

**Comment by debo19** [comment]
@Akshit42-hue You can take a look at this [PR](https://github.com/meshery/play/pull/13). We should use a similar approach to change the radio buttons to a select option here as well.

**Comment by Akshit42-hue** [comment]
Thanks for the lead @debo1, but the point is how can I check the role gets assigned

**Comment by debo19** [comment]
@Akshit42-hue this should work. Once you make all the changes, do a final test and let me know. I'll check the role and get back to you. Sounds good?

**Comment by Akshit42-hue** [comment]
yes this sounds good to me

> @Akshit42-hue this should work. Once you make all the changes, do a final test and let me know. I'll check the role and get back to you. Sounds good?
Yes, this sounds good to me

**Comment by Akshit42-hue** [comment]
Please do have a look @debo19

**Comment by l5io** [comment]
🚀 Preview for commit 549e77484a27d0eaae27f4d195258c3eaea1add7 at: https://62692a7c5f645716941d84d4--layer5ng.netlify.app

**Review by debo19 — CHANGES_REQUESTED** [review]
@Akshit42-hue It's still not working. Try using the `<Field />` tag instead.
 `<Field as="select" name="role">`

File: src/sections/Meshmap/index.js [hunk 1]
```diff
@@ -133,57 +132,25 @@ const Meshmap = () => {
...
-                            Student
-                        </label>
-                        <label>
-                          <Field type="radio" name="role" value="Other" />
-                            Other
-                        </label>
+                      <label htmlForFor="role" className="form-name">What role best identifies you?<span className="required-sign">*</span></label>
+                      <div role="group" className="formRight" aria-labelledby="select">
+                        <Field as="select" name="role">
+                          <option selected disabled hidden>Select your role</option>
```
Comment (debo19) [inline src/sections/Meshmap/index.js:138]:
```suggestion
                          <option>Select your role</option>
```

File: src/sections/Meshmap/index.js [hunk 2]
```diff
@@ -133,57 +132,25 @@ const Meshmap = () => {
...
+                          <option value="Architect" name="role">Architect</option>
```
Comment (debo19) [inline src/sections/Meshmap/index.js:139]:
```suggestion
                          <option value="Architect">Architect</option>
```

File: src/sections/Meshmap/index.js [hunk 3]
```diff
@@ -133,57 +132,25 @@ const Meshmap = () => {
...
+                          <option value="Business Operations" name="role">Business Operations</option>
```
Comment (debo19) [inline src/sections/Meshmap/index.js:140]:
```suggestion
                          <option value="Business Operations">Business Operations</option>
```

Comment (debo19) [reply src/sections/Meshmap/index.js:140]: Similarly, remove this for other options too.

**Review by debo19 — CHANGES_REQUESTED** [review]
It's submitting the `role` now. Good job! 
Can you make the arrow slightly bigger and increase the right padding?

**Comment by l5io** [comment]
🚀 Preview for commit d4c5edcefc27cbe9dff240a8188b8767d89cde6a at: https://626a16134c2628551fb67aee--layer5ng.netlify.app

**Comment by Akshit42-hue** [comment]
Required changes have been done please do have a look @debo19

File: src/sections/Meshmap/meshmap.style.js [hunk 4]
```diff
@@ -124,14 +124,33 @@ p.caption {
...
+          border-radius: 10px;
+          height: 50px;
+          padding: 1rem 1rem;
+          width: 100%;
+          font-size: 0.85rem;
+          margin: 0.5rem 0;
+
+         
+          
+          background: url(http://cdn1.iconfinder.com/data/icons/cc_mono_icon_set/blacks/16x16/br_down.png) no-repeat right #ddd;
```
Comment (debo19) [inline src/sections/Meshmap/meshmap.style.js:139]: Form is working now. Can we use a react icon instead of this external image here?

Comment (Akshit42-hue) [reply src/sections/Meshmap/meshmap.style.js:139]: For using the react icons I have to make changes in the dropdown list

**Comment by l5io** [comment]
🚀 Preview for commit 1320aa1c2a9c64a9d650f8869b5e573498b92fed at: https://6272120e86658947eca89ccf--layer5ng.netlify.app

**Review by debo19 — APPROVED** [review]
@Akshit42-hue Thank you seeing this through. 🎉

**Pull request merged**